spec/acceptance/tic_tac_toe_spec.rb

```diff
@@ -1,47 +1,57 @@
 # frozen_string_literal: true
 
-require 'store_game_state'
-require 'retrieve_game_state'
-require 'modify_game_state'
-require 'check_game_state'
+require 'save_game'
+require 'load_game'
+require 'make_move'
+require 'determine_outcome'
 require 'test_doubles/game_state_storage_gateway_fake'
 
 describe 'Tic Tac Toe' do
-  def given_a_new_game
-    game_state = [0, 0, 0, 0, 0, 0, 0, 0, 0]
-    store_game_state.execute(game_state)
-  end
-
+  
   let(:game_state_gateway) { GameStateStorageGatewayFake.new }
-  let(:store_game_state) do
-    StoreGameState.new(game_state_gateway: game_state_gateway)
+  let(:save_game) do
+    SaveGame.new(game_state_gateway: game_state_gateway)
   end
-  let(:retrieve_game_state) do
-    RetrieveGameState.new(game_state_gateway: game_state_gateway)
+  let(:load_game) do
+    LoadGame.new(game_state_gateway: game_state_gateway)
   end
-  let(:modify_game_state) do
-    ModifyGameState.new(game_state_gateway: game_state_gateway)
+  let(:make_move) do
+    MakeMove.new(game_state_gateway: game_state_gateway)
+  end
+  
+  def given_a_new_game
+    game_state = [0, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
Can initialise a new array by implementing either of the following:

`game_state = Array.new(9, 0)`
`game_state = [0] * 9`

I think it reads a lot nicer.

Reply:
You're right, I've changed the code to use the first suggestion, thanks 😄